src/pyconverter/xml2py/writer.py

```diff
@@ -295,90 +391,246 @@ def write_source(
         if not os.path.isdir(template_path):
             download_template()
 
+    new_package_name = get_config_data_value(config_path, "new_package_name")
     new_package_path = os.path.join(target_path, new_package_name)
 
     if clean:
         if os.path.isdir(new_package_path):
             shutil.rmtree(new_package_path)
 
-    cmd_path = os.path.join(new_package_path, GENERATED_SRC_CODE)
-    if not os.path.isdir(cmd_path):
-        os.makedirs(cmd_path)
+    library_path = get_library_path(new_package_path, config_path)
 
-    for ans_name, cmd_obj in tqdm(commands.items(), desc="Writing commands"):
-        if ans_name in SKIP_XML:
-            continue
-        cmd_name = cmd_map[ans_name]
-        path = os.path.join(cmd_path, f"{cmd_name}.py")
-        with open(path, "w", encoding="utf-8") as fid:
-            fid.write(cmd_obj.to_python(cmd_map, custom_functions))
+    if not os.path.isdir(library_path):
+        os.makedirs(library_path)
 
-        try:
-            exec(cmd_obj.to_python(cmd_map, custom_functions))
-        except:
-            raise RuntimeError(f"Failed to execute {cmd_name}.py") from None
+    module_name_list = []
 
-    mod_file = os.path.join(cmd_path, "__init__.py")
-    with open(mod_file, "w") as fid:
-        for ans_name in commands:
-            if ans_name in SKIP_XML:
+    if structured == False:
+        package_structure = None
+        for initial_command_name, command_obj in tqdm(command_map.items(), desc="Writing commands"):
+            if initial_command_name in SKIP_XML:
                 continue
-            cmd_name = cmd_map[ans_name]
-            fid.write(f"from .{cmd_name} import *\n")
-        fid.write("try:\n")
-        fid.write("    import importlib.metadata as importlib_metadata\n")
-        fid.write("except ModuleNotFoundError:\n")
-        fid.write("    import importlib_metadata\n\n")
-        fid.write("__version__ = importlib_metadata.version(__name__.replace('.', '-'))\n")
-        fid.write('"""PyConverter-GeneratedCommands version."""\n')
+            python_name = name_map[initial_command_name]
+            module_name_list.append(python_name)
+            path = os.path.join(library_path, f"{python_name}.py")
+            with open(path, "w", encoding="utf-8") as fid:
+                python_method = command_obj.to_python(custom_functions)
+                fid.write(f"{python_method}\n")
+            fid.close()
+
+            try:
+                exec(command_obj.to_python(custom_functions))
+            except:
+                raise RuntimeError(f"Failed to execute {python_name}.py") from None
```

Comment:
This is very bad practice.

1. You do not use bare exceptions.
2. If you do, raising ``from None`` hides all the exception information.

If you must do bare exceptions, you should do:

```suggestion
            except Exception as e:
                raise RuntimeError(f"Failed to execute {python_name}.py") from e
```

Ref: https://stackoverflow.com/a/46091127/6650211